src/main/java/com/rapid7/client/dcerpc/mssrvs/messages/NetprPathCanonicalizeRequest.java

```diff
@@ -161,8 +161,10 @@ public NetprPathCanonicalizeRequest(String serverName, String pathName, int outB
     public void marshal(final PacketOutput stubOut) throws IOException {
         stubOut.writeStringRef(serverName, true);
         stubOut.writeString(pathName, true);
```

Comment:
You'll need to align after `writeStringRef` as well, since `serverName` is variable. We've just been lucky so far as we've used an empty string:
```
stubOut.writeStringRef(serverName, true);
stubOut.align();
stubOut.writeString(pathName, true);
```

Reply:
Good catch. I didn't realise I was fixing this method too.